BonApp.js

```diff
@@ -47,306 +45,305 @@ render: function(output) {
-          return $('<li>').append('<a>' +
+          return $('<li>').append('<div>' +
                                   '<a id=\'search-label\'>' + item.label + '</a>' +
-                                  '<a id=\'search-description\'>' + item.desc + '</a>' + '</a>')
-                          .appendTo(ul);
-        };
-      });
-    };
+                                  '<a id=\'search-description\'>' + item.desc + '</a>' + '</div>')
+                          .appendTo(ul)
+        }
+      })
+    }
 
 
     var cleanUpStationLabel = function(theStation) {
-      var cleanString = theStation.replace('@', '');
-      var capitalizedStr = capitalizeStr(cleanString, 8, 9);
-      capitalizedStr = '<strong>' + capitalizedStr + '</strong>';
+      var cleanString = theStation.replace('@', '')
+      var capitalizedStr = capitalizeStr(cleanString, 8, 9)
+      capitalizedStr = '<strong>' + capitalizedStr + '</strong>'
 
-      return capitalizedStr;
-    };
+      return capitalizedStr
+    }
 
 
     var updateView = function(dom, theDate, widgetSelect, title, outputString) {
-      var err;
-      var theLastUpdate = lastUpdated;
+      var err
+      var theLastUpdate = lastUpdated
 
       try {
-        dom.find(widgetSelectTitle).html(widgetSelect);
+        $(document).find(widgetSelectTitle).html(widgetSelect)
       }
       catch (_error) {
-        err = _error;
-        console.log('Error: "' + err.message + '" on line ' + err.line);
+        err = _error
+        console.log('Error: "' + err.message + '" on line ' + err.line)
       }
       try {
-        dom.find(widgetTitle).html(title);
+        $(document).find(widgetTitle).html(title)
       }
       catch (_error) {
-        err = _error;
-        console.log('Error: "' + err.message + '" on line ' + err.line);
+        err = _error
+        console.log('Error: "' + err.message + '" on line ' + err.line)
       }
       try {
-        dom.find(date).html(theDate);
+        $(document).find(date).html(theDate)
       }
       catch (_error) {
-        err = _error;
-        console.log('Error: "' + err.message + '" on line ' + err.line);
+        err = _error
+        console.log('Error: "' + err.message + '" on line ' + err.line)
       }
       try {
-        dom.find(footerBA).html(theLastUpdate);
+        $(document).find(footerBA).html(theLastUpdate)
       }
       catch (_error) {
-        err = _error;
-        console.log('Error: "' + err.message + '" on line ' + err.line);
+        err = _error
+        console.log('Error: "' + err.message + '" on line ' + err.line)
       }
       try {
-        return dom.find(food).html(outputString);
+        return $(document).find(food).html(outputString)
       }
       catch (_error) {
-        err = _error;
-        console.log('Error: "' + err.message + '" on line ' + err.line);
+        err = _error
+        console.log('Error: "' + err.message + '" on line ' + err.line)
```

Comment:
Just move all of these try/catches into one. They do the exact same thing to handle the error, so there's no point to having them separate.

Reply:
Fixed. Good idea.